packages/x-data-grid/src/hooks/features/clipboard/useGridClipboard.ts

```diff
@@ -119,7 +119,17 @@ export const useGridClipboard = (
     [apiRef, ignoreValueFormatter, clipboardCopyCellDelimiter],
   );
 
-  useGridNativeEventListener(apiRef, apiRef.current.rootElementRef!, 'keydown', handleCopy);
+  useGridApiOptionHandler(apiRef, 'rootMount', () => {
+    const element = apiRef.current.rootElementRef.current;
+    if (!element) {
+      return undefined;
+    }
+    element.addEventListener('keydown', handleCopy);
+
+    return () => {
+      element.removeEventListener('keydown', handleCopy);
+    };
+  });
```

Comment:
Would it make sense to rewrite `useGridNativeEventListener` to match this codeblock? I think it's not used anymore with the changes in this PR.

Reply:
Hmm, either that or remove `useGridNativeEventListener` altogether 🤔

Reply:
I'll update `useGridNativeEventListener`, I just realized that I made a mistake with unsubscriptions.